pkg/app/app.go

```diff
@@ -1352,6 +1352,14 @@ Do you really want to apply?
 	if !interactive || interactive && r.askForConfirmation(confMsg) {
 		r.helm.SetExtraArgs(argparser.GetArgs(c.Args(), r.state)...)
 
+		for _, release := range st.Releases {
```

Comment:
@Sajfer Thanks a lot for the fix! Juts to be extra sure, you're triggering preapply events also for unprocessed(=not-deleted and not-updated releases without any changes to be applied) releases. Is it intentional? If not, I'm prepared for fixing it in a way I've summarized in my previous comment.

Reply:
Sure, go ahead, it was not intended.